src/libPersistence/BlockStorage.cpp

```diff
@@ -417,6 +417,36 @@ bool BlockStorage::ResetDB(DBTYPE type)
     return ret;
 }
 
+std::vector<std::string> BlockStorage::GetDBName(DBTYPE type)
+{
+    std::vector<std::string> ret;
+    switch (type)
+    {
+    case META:
+        ret.push_back(m_metadataDB.GetDBName());
```

Comment:
Do u need break after this?

Reply:
Still need @kaikawaliu inputs on this. I still likes after each condition, there should be a break.

Reply:
ok